components/GetBorderlessBox.coffee

```diff
@@ -37,7 +37,7 @@ isBorder = (array, prev, threshold) ->
 
 exports.getComponent = ->
   c = new noflo.Component
-  c.description = 'Extract the borderless bounding box of an image'
+  c.description = 'Extract a bounding box with top and bottom black borders removed (avoids removing more than 50% of image)'
```

Comment:
Suggestion for description: `Extract bounding box with top and bottom borders removed, respecting area threshold`